### PR TITLE
Fix a bug in artnet universe mapping

### LINKED ISSUE #856: Can't set Artnet universe 15
There seems to be a bug in the Artnet mapping of universes.
If I create an Artnet output with universe 15, it will show up as 0:0:0
From what I understand it should show up as 0:0:15
Then to address universe 16 you would set the subnet to 1 instead of 0...

### PR DESCRIPTION
**Pull request by nomis52**
Closes #856

### CODE REVIEW

File: plugins/artnet/ArtNetPort.cpp [hunk 1]
```diff
@@ -35,10 +35,15 @@ using ola::rdm::RDMCommand;
 using std::string;
 using std::vector;
 
+namespace {
+static const uint8_t ARTNET_UNIVERSE_COUNT = 16;
+};  // namespace
+
 void ArtNetInputPort::PostSetUniverse(Universe *old_universe,
                                       Universe *new_universe) {
   if (new_universe)
```
Comment (peternewman) [inline plugins/artnet/ArtNetPort.cpp]: {} please

File: plugins/artnet/ArtNetPort.cpp [hunk 2]
```diff
@@ -139,7 +144,7 @@ void ArtNetOutputPort::PostSetUniverse(Universe *old_universe,
                                        Universe *new_universe) {
   if (new_universe)
```
Comment (peternewman) [inline plugins/artnet/ArtNetPort.cpp]: Again

File: plugins/artnet/ArtNetPort.cpp [hunk 3]
```diff
@@ -35,10 +35,15 @@ using ola::rdm::RDMCommand;
...
 using std::vector;
 
+namespace {
+static const uint8_t ARTNET_UNIVERSE_COUNT = 16;
+};  // namespace
+
 void ArtNetInputPort::PostSetUniverse(Universe *old_universe,
                                       Universe *new_universe) {
   if (new_universe)
-    m_node->SetOutputPortUniverse(PortId(), new_universe->UniverseId() % 0xf);
```
Comment (peternewman) [inline plugins/artnet/ArtNetPort.cpp]: ArtNetNode uses (port->universe_address & 0xf) for stuff, we should probably switch to the same constant throughout and ideally use the same method of processing.

File: plugins/artnet/ArtNetPort.cpp (hunk 1 above)
Comment (nomis52) [inline plugins/artnet/ArtNetPort.cpp]: Done

File: plugins/artnet/ArtNetPort.cpp (hunk 2 above)
Comment (nomis52) [inline plugins/artnet/ArtNetPort.cpp]: Done

**Comment by peternewman** [comment]
Looks good, just a query regarding matching behaviour with ArtNetNode

**Comment by nomis52** [comment]
I think that's out of scope for this change.

**Comment by peternewman** [comment]
Okay.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**